File: notify/v1/binding/create-binding-server-apns/create-binding-server-apns.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/console
$sid    = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$binding = $twilio->notify->v1->services("ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                              ->bindings
                              ->create("apn_device_token",
                                       "apn",
                                       "00000001",
                                       array(
                                           'endpoint' => "endpoint_id",
                                           'tag' => "preferred device"
                                       )
                              );

print($binding.sid);

File: notify/v1/binding/create-binding-server-apns/create-binding-server-apns.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Notify.V1.Service;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var tag = new List<string> {
            "preferred device"
        };

        var binding = BindingResource.Create(
            endpoint: "endpoint_id",
            tag: tag,
            identity: "00000001",
            bindingType: BindingResource.BindingTypeEnum.Apn,
            address: "apn_device_token",
            pathServiceSid: "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(binding.Sid);
    }
}
